devblog/2024-03-11-updated-antenna-denylist-classifier/index.mdx

```diff
@@ -0,0 +1,102 @@
+---
+layout: post
+title: Updated antenna denylist classifier
+date: 2024-03-11 09:00 -0800
+hide_table_of_contents: true
+authors: [vagabond]
+---
+
+# Updated antenna denylist classifier
+
+Back in January, we added two new denylist classifiers, the antenna split
+classifier and the latency classifier. In the time since, we've identified some
+shortcomings with the antenna classifier, so we've put in some time improving
+it.
+
+The old classifier was rather crude in that we only ever configured it to look
+for reciprocal (in both directions) links where the RSSI was above 0db. It was
+difficult to extend it beyond that because the classifier didn't have access to
+some information about how powerful the signal was so it could not properly
+calculate the attentuation. We've since improved that by adding the missing
+information and we've been testing a significantly improved version of the
+classifier for the last few weeks. During the last few IoT working group
```

Comment:
Doesn't age well.. 

The results were presented over a number of IoT working group sessions, and on February 4th, 2024, the working group concluded we should move ahead with enabling this feature as of the March 11th deny list.